### DISCUSSION TITLE
Updating from Big Sur to Ventura causes machine to bootloop

### DISCUSSION THREAD

**Discussion by eversiege** [Coopydood/ultimate-macOS-KVM#9] [General]
I am not pushing this as an issue, as it's clearly not your fault, but I (and some of my friends) have tried to update Big Sur to Ventura, resulting the machine to hard brick/bootloop. The picture below shows the "Kernel panic" after updating. 

![image](https://github.com/Coopydood/ultimate-macOS-KVM/assets/69052669/287b029e-0c80-4cac-84df-04b196bae05c)

**Comment by Coopydood** [comment]
> I am not pushing this as an issue, as it's clearly not your fault, but I (and some of my friends) have tried to update Big Sur to Ventura, resulting the machine to hard brick/bootloop. The picture below shows the "Kernel panic" after updating. 
> 
> 
> 
> ![image](https://github.com/Coopydood/ultimate-macOS-KVM/assets/69052669/287b029e-0c80-4cac-84df-04b196bae05c)
> 
> 

I'll have a look into this.

**Comment by Coopydood** [comment]
Converted to [this issue](https://github.com/Coopydood/ultimate-macOS-KVM/issues/10).
